### ISSUE TITLE
Make database backup possible

### ISSUE THREAD

**Issue by jvdzwaan** [TICCLAT/ticclat#24]
Make cronjob to backup every week?
after we reingested all data

**Comment by jvdzwaan** [comment]
It is good to make regular backups of the ticclat database, as reingesting data costs time.

**Comment by egpbos** [comment]
This seems to fit: https://www.linode.com/docs/databases/mysql/use-mysqldump-to-back-up-mysql-or-mariadb/

**Comment by egpbos** [comment]
This looks even nicer: https://medium.com/@mhagemann/how-to-backup-mysql-databases-automatically-on-ubuntu-17-10-a2b29fb47ac9

**Comment by egpbos** [comment]
I set up automysqlbackup on the VM.

Compared to the default configuration (in `/etc/default/automysqlbackup`), I set `DBHOST=127.0.0.1` (Ubuntu's MySQL configuration is really particular about `localhost` vs `127.0.0.1`) and I changed the PASSWD command to grep the password from `~/mysql_login.cnf`; in that file (with `chmod 600`) I put the root password plaintext, which obviously is not such a great deployment solution (indeed, the program dutifully warns about this in the ERRORS file in the backup directory). I also set `BACKUPDIR="/data/automysqlbackup"`.

Then I added a root cronjob with `sudo crontab -e`:

```cron
@daily automysqlbackup
```

I manually ran the tool once from the command line with simply

```sh
sudo automysqlbackup
```

This took a couple of minutes (ca. 15 minutes for the ticclat production database), but now we indeed have zipped backups of all the databases in `/data/automysqlbackup`.
